modules/appeals_api/spec/factories/higher_level_reviews.rb

```diff
@@ -52,14 +52,14 @@
   factory :extra_higher_level_review_v0,
           class: 'AppealsApi::HigherLevelReview', parent: :extra_higher_level_review_v2 do
     api_version { 'V0' }
-    auth_headers { FixtureHelpers.fixture_as_json 'higher_level_reviews/v0/valid_200996_headers_extra.json' }
+    auth_headers { FixtureHelpers.fixture_as_json 'higher_level_reviews/v0/valid_200996_headers.json' }
     form_data { FixtureHelpers.fixture_as_json 'higher_level_reviews/v0/valid_200996_extra.json' }
   end
 
   factory :minimal_higher_level_review_v0,
           class: 'AppealsApi::HigherLevelReview', parent: :minimal_higher_level_review_v2 do
     api_version { 'V0' }
-    auth_headers { FixtureHelpers.fixture_as_json 'higher_level_reviews/v0/valid_200996_headers_minimum.json' }
```

Comment:
Because headers have been pared down in this PR, the minimal values would now be an empty object